cfgov/cfgov/urls.py

```diff
@@ -597,14 +597,48 @@ def empty_200_response(request, *args, **kwargs):
 urlpatterns = urlpatterns + category_redirects
 
 if settings.ALLOW_ADMIN_URL:
-    patterns = [
-        re_path(r'^login/$', login_with_lockout, name='cfpb_login'),
+    # If SAML2 auth is enabled, /login will redirect to /saml2/login,
+    # which in turn will redirect to the configured identity provider.
+    if settings.SAML_AUTH:
```

Comment:
To avoid adding this complexity to this file, what do you think about creating some other auth-specific urls.py, perhaps in a separate auth-specific app? We could then move the various v1 auth-specific views in there as well.

Reply:
I like that idea a lot, acutally.

Reply:
Moved in 33e1ae1!